### PR TITLE
fix(simple buy): use maxPossibleOrder instead of account balance for sb FUNDS orders

### PR DESCRIPTION
**Pull request by blockdylanb**
## Description (optional)
When a user would click the max buy on a simple buy order from their FUNDS the form would use the max available funds in the account instead of using the `maxPossibleOrder` limit we get from the /trades/limits endpoint

## Testing Steps (optional)
- put funds in your Gold account
- open simple buy and click max buy, if your funds are less than 25,000 it should put in your total available funds, if they are over 25,000 it should input 20,000 (max limit per transaction)

Test scenarios still needed
- [x] funds under 20k
- [x] silver tier user
- [x] different payment methods

### CODE REVIEW

**Comment by yichok** [comment]
Ллл

**Review by yichok — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/selectors.ts [hunk 1]
```diff
@@ -19,10 +18,9 @@ import {
 } from '../exchange/services'
 import { getInputFromPair, getOutputFromPair } from '../swap/model'
 import { getRate } from '../swap/utils'
+import { LIMIT } from './model'
 import { SBCardStateEnum, SBCheckoutFormValuesType } from './types'
 
-const SDD_LIMIT = { min: '500', max: '10000' } as SDDLimits
-
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/selectors.ts:25]: cool 👏

**Review by milan-bc — APPROVED** [review]
I tested this one it works as expected, also code LGTM.
We should add TODO to cleanup code and use one single function for calling limits API now we have 2 on 2 diff spots in swap and in SB.

**Comment by ma2356748** [comment]
لماذا لم تتم عمليات السحب الي محفظتي

File: packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/selectors.ts (hunk 1 above)
Comment (blockdylanb) [reply packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/selectors.ts:25]: I moved several consts to the model file in the simple buy data dir to clean up the template files a bit.

**Comment by ma2356748** [comment]
هل يوجد بالفعل رصيد حساب فعلي

في الأربعاء، ٣ فبراير ٢٠٢١ ١:٣٨ ص blockdylanb <notifications@github.com>
كتب:

**Comment by ma2356748** [comment]
Please send  1 BTC to the Bitcoin address
1E8QUZsg7XCGYTan3paF3FhdmM35x9G48e.

bitcoin://1E8QUZsg7XCGYTan3paF3FhdmM35x9G48e?amount=1

**Pull request merged**